src/Symfony/Component/Validator/Tests/Constraints/AbstractConstraintValidatorTest.php

```diff
@@ -244,7 +244,7 @@ protected function expectValidateValueAt($i, $propertyPath, $value, $constraints
 
     protected function assertNoViolation()
     {
-        $this->assertCount(0, $this->context->getViolations());
+        $this->assertSame(0, $violationsCount = count($this->context->getViolations()), sprintf('0 violation expected. Got %d.', $violationsCount));
```

Comment:
Wouldn't it be enough to add a message as the third parameter when calling `assertCount()`?